### ISSUE TITLE
how to pass argument to load method

### ISSUE THREAD

**Issue by fbucek** [cksac/dataloader-rs#26]
I am using async_graphql and dataloader and it works fine.

But when I want to limit output using `last` argument. I do not know how to do this with dataloader.

```graphql
authors {
 name
 books(last: 30)
}
```

Getting books is called like this in graphql object ( without data loader ) 

```rust
pub async fn books(
    &self,
    ctx: &Context<'_>,
    last: i32,
) -> Result<Vec<Books>, AppError> {
    ctx.data_unchecked::<BooksRepository>()
        .get_for_id(self.id, last)
        .await
}
```

When using dataloader I do not know how to make it work.

```rust
pub async fn books(
    &self,
    ctx: &Context<'_>,
    last: i32,
) -> Result<Vec<books>, AppError> {
    let loader = ctx.data_unchecked::<BooksLoader>();
    loader.load(self.id).await
}
```

Maybe I just overlooked something or just do not have mu day, but I do not know how to solve it. Thanks for suggestion.

**Comment by cksac** [comment]
You can implement with other key type, e.g LoadFn<(usize, usize)>

**Comment by fbucek** [comment]
Thanks for suggestion, I have tried Arc<Mutex<Config>> mutex, so I can set variables for method in struct which implements BatchFn, but then I have realized that I do not know how to do SQL query to return for X items last X attributes. 

For example for last 10 authors get last 10 books. Now it is 11 queries, but I do not know how to make it in 2 queries in SQL regardless of dataloader.

I am closing, thanks for advice. I will try it when I solve SQL problem.
